### ISSUE TITLE
*.rpt Link Flow Summary wrong column for weir results in GUI

### ISSUE THREAD

**Issue by mel-meng** [USEPA/Stormwater-Management-Model#36]
This is possible bug in the EPA SWMM 5.1 GUI. 
After running the model below, go to "Report->Summary->Topic: Link Flow"
[model_full_features.inp](https://github.com/OpenWaterAnalytics/pyswmm/blob/master/pyswmm/tests/data/model_full_features.inp
)

For the weir "C3" the last value 0.47 should be the last column. Instead it is displayed as max. vel.

A review of the *.rpt file showed that, for weir, the values of "max. vel" and "max. full flow" were omitted in the the line. A quick check the of "Tokenize" procedure in "Uutils.pas line 889 showed that the routine doesn't consider such situation, it assumes no values are omitted.

```
  ********************
  Link Flow Summary
  ********************
  
  -----------------------------------------------------------------------------
                                 Maximum  Time of Max   Maximum    Max/    Max/
                                  |Flow|   Occurrence   |Veloc|    Full    Full
  Link                 Type          CFS  days hr:min    ft/sec    Flow   Depth
  -----------------------------------------------------------------------------
  C1:C2                CONDUIT      7.22     2  09:59     18.36    0.90    0.50
  C2                   PUMP         9.41     0  10:00                          
  C3                   WEIR        12.12     0  10:00                      0.47
```

**Comment by mel-meng** [comment]
![swmm_rpt_error](https://user-images.githubusercontent.com/2023246/77543553-432a4600-6e7e-11ea-9c24-7605316fafd3.png)

**Comment by dickinsonre** [comment]
Or course this is a Delphi reading problem with the RPT file.  In the actual report file, it does seem like the weir depth is in the right place.  I would test the other conduit types such as outlets and orifices to see if there is a similar problem.  It is a table generation issue in Delphi and not an engine problem.

**Comment by cbuahin** [comment]
Addressed in #56 .
